templates/content.php

```diff
@@ -1,9 +1,63 @@
-<article <?php post_class(); ?>>
-  <header>
-    <h2 class="entry-title"><a href="<?php the_permalink(); ?>"><?php the_title(); ?></a></h2>
-    <?php get_template_part('templates/entry-meta'); ?>
-  </header>
-  <div class="entry-summary">
-    <?php the_excerpt(); ?>
-  </div>
-</article>
+<div class="layout-container full--until-large">
+  <div class="flex-container cf">
+    <div class="shift-left--fluid column__primary bg--white no-pad--top no-pad--btm can-be--dark-light">
+      <?php if (!have_posts()) : ?>
+        <div class="pad--primary no-pad--top no-pad--btm spacing--half text">
+          <div class="alert alert-warning pad-double--top pad-half--btm">
+            <?php _e('Sorry, no results were found.', 'sage'); ?>
+          </div>
+          <?php get_template_part('patterns/components/search-form'); ?>
+        </div>
+      <?php endif; ?>
+      <div class="spacing--half">
+        <h2 class="font--tertiary--l theme--primary-text-color pad pad-double--top pad-half--btm">
+          <?php if(is_home()): ?>
+            All News
+          <?php elseif(is_category()): ?>
+            <?php $categories = get_the_category(); ?>
+            <?php if (!empty($categories)): ?>
+                All <?php echo esc_html($categories[0]->name); ?>
+            <?php endif; ?>
+          <?php endif; ?>
+        </h2>
+        <hr>
+      </div>
+      <div class="g g-3up--at-medium with-divider grid--uniform">
+        <?php
+          $news = array(
+            'cat' => array(14)
+          );
+        ?>
+        <?php if(is_home()): query_posts($news); ?>
+          <?php while (have_posts()) : the_post(); ?>
+            <div class="gi">
+              <div class="spacing">
+                <div class="pad">
+                  <?php
+                    $title = get_the_title();
+                    $intro = get_field('intro');
+                    $body = strip_tags(get_the_content());
+                    $excerpt_length = 100;
+                    $image = get_post_thumbnail_id();
+                    $kicker = '';
```

Comment:
For items that you have to make `= ''`, you can do a check for it where the variable is being printed so you don't have to do this for each instance. 

```php
<?php if (isset($kicker)): ?> A kicker exists <?php endif; ?>
```
OR
```php
<?php echo (isset($kicker) ? $kicker : ''); ?>
```